crates/control/src/motion/look_at.rs

```diff
@@ -67,12 +66,8 @@ impl LookAt {
             None => return default_output,
         };
 
-        let ground_to_robot = match context.ground_to_robot {
-            Some(ground_to_robot) => *ground_to_robot,
-            None => return default_output,
-        };
```

Comment:
Why did you remove this?

Reply:
This is fixed, there was a mistake in our thinking.